src/modules/dashboard/components/ActionsPage/TotalStakeWidget/TotalStakeWidget.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import formatNumber from 'format-number';
+
+import Heading from '~core/Heading';
+import ProgressBar from '~core/ProgressBar';
+import { useLoggedInUser } from '~data/index';
+import {
+  useStakeAmountsForMotionQuery,
+  useColonyNativeTokenQuery,
+  useTokenInfoLazyQuery,
+} from '~data/generated';
+import { Address } from '~types/index';
+
+import styles from './TotalStakeWidget.css';
+
+const displayName = 'TotalStakeWidget';
+
+type Props = {
+  tokenDecimals: number;
+  colonyAddress: Address;
+  motionId: string;
+  isObjectionStake?: boolean;
+};
+
+const MSG = defineMessages({
+  motionTitle: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.motionTitle',
+    defaultMessage: 'Stake',
+  },
+  objectionTitle: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.objectionTitle',
+    defaultMessage: 'Goal',
+  },
+  stakeProgress: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.stakeProgress',
+    defaultMessage: '{totalPercentage}% of {requiredStake} {tokenSymbol}',
+  },
+  userStake: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.userStake',
+    defaultMessage: `You staked {userPercentage}% of this motion ({userStake} {tokenSymbol}).`,
+  },
+});
+
+const TotalStakeWidget = ({
+  tokenDecimals,
+  colonyAddress,
+  motionId,
+  isObjectionStake = false,
+}: Props) => {
+  const { walletAddress } = useLoggedInUser();
+  const { data } = useStakeAmountsForMotionQuery({
+    variables: {
+      colonyAddress,
+      userAddress: walletAddress,
+      motionId,
+      isObjectionStake,
+      tokenDecimals,
+    },
+  });
+  const {
+    totalStaked,
+    userStake,
+    requiredStake,
+  } = data?.stakeAmountsForMotion || {
+    totalStaked: 0,
+    userStake: 0,
+    requiredStake: 0,
+  };
+  const {
+    data: nativeTokenAddressData,
+    loading: loadingNativeTokenAddress,
+  } = useColonyNativeTokenQuery({
+    variables: { address: colonyAddress },
+  });
+  const [
+    fetchTokenInfo,
+    { data: tokenInfoData, loading: loadingTokenInfoData },
+  ] = useTokenInfoLazyQuery();
+
+  useEffect(() => {
+    if (nativeTokenAddressData) {
+      const {
+        processedColony: { nativeTokenAddress },
+      } = nativeTokenAddressData;
+      fetchTokenInfo({ variables: { address: nativeTokenAddress } });
+    }
+  }, [fetchTokenInfo, nativeTokenAddressData]);
+
+  const totalStakedPercentage = (totalStaked * 100) / (requiredStake || 1);
+  const userStakePercentage = (userStake * 100) / (requiredStake || 1);
+  const formattedTotalStakedPercentage = formatNumber({
+    truncate: 2,
+  })(totalStakedPercentage);
+  const formattedUserStakePercentage = formatNumber({
+    truncate: 2,
+  })(userStakePercentage);
+
+  return (
+    <div>
+      <div className={styles.widgetHeading}>
+        <Heading
+          appearance={{
+            theme: 'dark',
+            size: 'small',
```

Comment:
I can see that in Figma specs the Heading is 13px, but when I inspect it, it's only 11px.

Reply:
Mmmm this heading should actually be `12px` which I think is the closest thing we have to `13px` in Heading sizes.

The one who is actually `11px` is the `span` below this Heading that displays the stake progress in text form

Reply:
I am not sure what you mean exactly. You can change the heading size manually to 13px if the specs say 13...